detect_secrets/plugins/keyword.py

```diff
@@ -153,22 +153,22 @@ def __init__(self, keyword_exclude=None,
 
     def analyze_string_content(self, string, line_num, filename):
         output = {}
-            if (
-                self.keyword_exclude
-                and self.keyword_exclude.search(string)
-            ):
-                return output
-            for identifier in self.secret_generator(
-                string,
-                filetype=determine_file_type(filename),
-            ):
-                secret = PotentialSecret(
-                    self.secret_type,
-                    filename,
-                    identifier,
-                    line_num,
-                )
-            output[secret] = secret
+        if (
+            self.keyword_exclude
+            and self.keyword_exclude.search(string)
+        ):
+            return output
+        for identifier in self.secret_generator(
+            string,
+            filetype=determine_file_type(filename),
+        ):
+            secret = PotentialSecret(
+                self.secret_type,
+                filename,
+                identifier,
+                line_num,
+            )
+        output[secret] = secret
```

Comment:
`output[secret] = secret` was on the right indentation level, my mistake

Reply:
Okay. Reverted it.